src/ess/polarization/base.py

```diff
@@ -435,16 +440,20 @@ def direct_beam_ratio(
 
 
 def he3_transmission(
-    opacity: He3OpacityFunction[Cell],
+    opacity_function: He3OpacityFunction[Cell],
     polarization_function: He3PolarizationFunction[Cell],
     transmission_empty_glass: He3TransmissionEmptyGlass[Cell],
-) -> He3Transmission[Cell]:
+) -> He3TransmissionFunction[Cell]:
```

Comment:
we should call it He3TransmissionFunctionFullPolarized (different function as above)

Reply:
Here add the T(p/m,cell) function for a full polarized beam (using both 3He cells), desribed under 'Spin-dependent transmission values'

Reply:
(nothing to fit, only insert already computed values)